docs/java-rest/high-level/indices/create_index.asciidoc

```diff
@@ -55,6 +55,13 @@ include-tagged::{doc-tests-file}[{api}-mappings-xcontent]
 <1> Mapping source provided as an `XContentBuilder` object, the Elasticsearch
 built-in helpers to generate JSON content
 
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-mappings-shortcut]
```

Comment:
I looked through the integration tests, however, there is no `// tag::create-index-request-mappings-shortcut` tag in the file, so this source include would not work.